shell/common/engine.cc

```diff
@@ -289,7 +289,15 @@ void Engine::SetViewportMetrics(const ViewportMetrics& metrics) {
   bool dimensions_changed =
       viewport_metrics_.physical_height != metrics.physical_height ||
       viewport_metrics_.physical_width != metrics.physical_width ||
-      viewport_metrics_.device_pixel_ratio != metrics.device_pixel_ratio;
+      viewport_metrics_.device_pixel_ratio != metrics.device_pixel_ratio ||
+      viewport_metrics_.physical_view_inset_top !=
```

Comment:
Is this necessary? The dimensions of the view itself aren't changing. It's only that the application now has to account for additional insets like notches and such. The new metrics will get delivered to the application no matter what.

Reply:
It wasn't needed. Removed.